_scripts/build-storybooks.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash -ex
+
+# TODO: only build if the current commit has changes to packages with storybooks - see modules-to-test.js for some hints?
+
+# Config from env vars with defaults
+SKIP_STORYBOOK_BUILD="${SKIP_STORYBOOK_BUILD:-false}"
+PROJECT_REPO="${PROJECT_REPO:-mozilla/fxa}"
+STORYBOOKS_REPO="${STORYBOOKS_REPO:-lmorchard/fxa-storybooks}"
+STORYBOOKS_URL="${STORYBOOKS_URL:-https://lmorchard.github.io/fxa-storybooks/}"
+STORYBOOKS_BRANCH="${STORYBOOKS_BRANCH:-gh-pages}"
+PUBLISH_ROOT="${PUBLISH_ROOT:-storybooks-publish}"
+MAX_STORYBOOK_AGE="${MAX_STORYBOOK_AGE:-90}"
+
+# Context for current build
+COMMIT_HASH=$(git rev-parse HEAD)
+
+# Build all the storybooks
+if [ "$SKIP_STORYBOOK_BUILD" == false ]; then
+    # fxa-payments-server relies on fxa-content-server .scss styles, which in turn
+    # rely on some modules in package.json
+    pushd ./packages/fxa-content-server
+    # npm ci
+    popd
+
+    STORYBOOKS=$(find packages -maxdepth 2 -type d -name '.storybook');
```

Comment:
This must be `STORYBOOKS=$(find . packages -maxdepth 2 -type d -name '.storybook');` for me in OS X.

Reply:
I *think* this find was okay, but further down the problem was `for STORYBOOK_DIR in $(find -type d -name 'storybook-static')` - which should be fixed in the commit I just pushed

Reply:
Ah, whoops, you are correct👌